fabfile/setup.py

```diff
@@ -223,19 +190,17 @@ def existing():
     add_fab_path_to_bashrc()
     git_repo = raw_input("Enter the git repository link\n(i.e. git@github.com:mr_programmer/robot_repository.git):\t")
     project_name = get_project_name_from_repo(git_repo)
+    repo_name = get_project_name_from_repo(git_repo, False)
     env.settings = {
         'git_repo': git_repo,
         'project_name': project_name
     }
     local("git clone {}".format(git_repo))
-
-    execute(create_ansible_env)
+    local("mv ./{} ./{}".format(repo_name, project_name))
     with bash():
         local("cp $FAB_PATH/../orchestration/Vagrantfile ./")
     recursive_file_modify('./Vagrantfile', env.settings, is_dir=False)
-    with ansible():
-        local("vagrant plugin install vagrant-vbguest")
-        local("vagrant up")
+    local("vagrant up")
```

Comment:
We've been able to reduce the dependencies to just Fabric and Ansible (and their respective dependencies) and pycrypto-on-pypi for older/experimental versions of Python 2.7. I could make the root level requirements.txt file have these dependencies and just tell users to `pip install -r` them (so they don't miss anything).